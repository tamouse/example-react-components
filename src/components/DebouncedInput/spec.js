import React from "react"
import renderer from "react-test-renderer"

import { DebouncedInput } from "./index"

describe("DebouncedInput", () => {
  it("renders the DebouncedInput component", () => {
    const formControl = (props)=>(<input {...props} />)
    const onChange = jest.fn()
    const actual = renderer.create(
      <DebouncedInput formControl={formControl} onChange={onChange}/>
    )
    expect(actual).toMatchSnapshot()
  })
})
