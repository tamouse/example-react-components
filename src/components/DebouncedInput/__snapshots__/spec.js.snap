// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DebouncedInput renders the DebouncedInput component 1`] = `
<input
  onBlur={[Function]}
  onChange={[Function]}
  onKeyUp={[Function]}
/>
`;
